### ISSUE TITLE
ansible-lint default exclude_paths does not match with ansible-compat 25.0.0

### ISSUE THREAD

**Issue by gmuloc** [ansible/ansible-lint#4477]
##### Summary

symptom and preliminary analysis:
* our pipeline started failing yesterday on ansible-lint reporting issues in a `.ansible` folder created when we run `ansible-lint`. 
* ansibe-compat version 25.0.0 was released yesterday and it changes the cache directory from `.cache` to `.ansible` in this PR: https://github.com/ansible/ansible-compat/pull/439

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
ansible-lint 24.12.2 using ansible-core:2.17.6 ansible-compat:25.0.0 ruamel-yaml:0.18.5 ruamel-yaml-clib:0.2.7   
```
- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

1. Upgrade ansible-compat to 25.0.0
2. Run `ansible-lint` as before.

##### Desired Behavior

Ignore the new cache location

###### Workaround

we have added the following in our `.ansible-lint` and it circumvent the issue:

```
exclude_paths:
  - .cache/
  - .ansible/
```

##### Actual Behavior

errors are reported in `.ansible` when we have excluded them from the checks for the given rule (line length) in yamllint configuration for the normal location.

```
ansible-lint -v
...
Read documentation for instructions on how to ignore specific rule violations.

# Rule Violation Summary

349 yaml profile:basic tags:formatting,yaml

Failed: 349 failure(s), 0 warning(s) on 6313 files. Profile 'production' was required, but 'min' profile passed.
```

with the workaround

```
ansible-lint -v
...
Passed: 0 failure(s), 0 warning(s) on 3155 files. Profile 'production' was required, and it passed.
```

**Comment by yurnov** [comment]
Thanks, @gmuloc, you save me a tons of time for the investigation of failing CI

**Comment by k-c-p** [comment]
ansibe-compat 25.0.0 has been [yanked](https://github.com/ansible/ansible-compat/releases/tag/v25.0.0). Latest stable is back to 24.10.0.
